js/output/pjs/pjs-output.js

```diff
@@ -437,8 +437,7 @@ window.PJSOutput = Backbone.View.extend({
                     // figure out whether we hit this case.
                     var message = i18n._("Error: %(message)s",
                         {message: errors[errors.length - 1].message});
-                    // TODO(jeresig): Find a better way to show this
-                    this.output.$el.find(".test-errors").text(message).show();
```

Comment:
test-errors doesn't exist anymore (hasn't in a while). We do display this in a hopper message, I believe, but I figured might as well console.warn() it as well